README.md

```diff
@@ -80,3 +80,68 @@ Retrofit.Builder()
     - Nothing is easier than that. Just make sure that your custom exception implements the Loggable interface, and the rest is handled during the error conversion. Every RemoteException which implements the Loggable interface is guaranteed to be logged.
   - **I don’t want to handle all of the possible exceptions with `try {} catch() {}` outside the data layer.**
     - In this case, you can use the ResultWrapper from the error-handler-core together with the wrapToResult helper function, if you don’t already have such a mechanism in your project.
+    
+    
+
+## Error handling Recommendations with tips & tricks
+
+The following recommendation works best with MVVM architecture and a separate data layer and with coroutines.
```

Comment:
To avoid multiple _and_ words in the sentence, I would modify to this:
_The following recommendation works best with MVVM architecture, a separate data layer and with coroutines._